### PR TITLE
VS code: automatically watch ghcid.txt 

### PR DESCRIPTION
**Pull request by chrismwendt**
This restores the ability to get error diagnostics from an existing ghcid process that writes `ghcid.txt` to any subdirectory in the VS Code workspace.

Here's a screenshot of files from 2 separate Haskell projects, each file's errors are coming from a different ghcid process:

image

I also stripped the line previews and indentation from hovers:

**Before**

image

**After**

image

Resolves https://github.com/ndmitchell/ghcid/issues/239 by restoring watch functionality that was removed in https://github.com/ndmitchell/ghcid/pull/118 and improving on it a bit.

### CODE REVIEW

**Comment by chrismwendt** [comment]
@ndmitchell Does this look good to merge?

File: plugins/vscode/.vscode/launch.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
-            "args": ["--extensionDevelopmentPath=${workspaceRoot}" ],
+            "args": ["--extensionDevelopmentPath=${workspaceRoot}", "${workspaceRoot}/../../test/foo", "${workspaceRoot}/../../test/foo/Root.hs" ],
```
Comment (ndmitchell) [inline plugins/vscode/.vscode/launch.json:10]: Is this deliberate?

Comment (chrismwendt) [reply plugins/vscode/.vscode/launch.json:10]: Yes, the `../../test/foo` directory within this repo is very convenient for developing/debugging. I used it all the time when working on this PR. IMO it's a sensible default that can be changed pretty easily.

**Comment by chrismwendt** [comment]
:+1: Added myself as an author.

**Comment by ndmitchell** [comment]
Great. Do you have any additional changes incoming, or shall I push this to the extension store? (will probably be in the next 2 days)

**Comment by chrismwendt** [comment]
No additional changes incoming, thanks for merging 🙇‍♂️

**Pull request merged**